db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/ValueRow.java

```diff
@@ -63,7 +64,7 @@
 	//	STATE
 	//
 	///////////////////////////////////////////////////////////////////////
-
+	private static final long serialVersionUID = 1626095185637654111l;
```

Comment:
This might cause serialization issues if upgrading, please check that's not the case or remove this.